packages/edge-login-ui-rn/src/components/common/MaterialInput.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import * as React from 'react'
-import { TextField } from 'react-native-material-textfield'
+import { TextField } from 'react-native-material-textfield-legacy'
```

Comment:
Why was this changed? Why was the new package added as a dependency in the first place?

Reply:
Because previous "react-native-material-textfield" is an old fork with additional commit but I need to use more fresh version of original repo. I left it for an old code that continue to use this fork. After updating the entire UI to a new one, it will be removed.